sortinghat/core/jobs.py

```diff
@@ -219,6 +219,49 @@ def recommend_matches(ctx, source_uuids, target_uuids, criteria, verbose=False):
     return job_result
 
 
+@django_rq.job
+def recommend_gender(ctx, uuids, api_token=None):
```

Comment:
I think the API token should be defined in the configuration of SortingHat instead of giving it in the call.